corehq/apps/reports/standard/monitoring.py

```diff
@@ -1062,8 +1062,8 @@ class WorkerActivityReport(WorkerMonitoringCaseReportTableBase, DatespanMixin):
     is_cacheable = True
 
     fields = [
-        'corehq.apps.reports.dont_use.fields.MultiSelectGroupField',
-        'corehq.apps.reports.dont_use.fields.UserOrGroupField',
+        'corehq.apps.reports.filters.select.MultiGroupFilter',
+        'corehq.apps.reports.filters.users.UserOrGroupFilter',
```

Comment:
probably not a big deal, but i remember looking into this for a second and noticing that the new filter doesn't have a default selected for the group. so it might slightly change functionality